docs/frameworks/dapr.md

```diff
@@ -1,34 +1,53 @@
+[Distributed Application Runtime (Dapr)](https://docs.dapr.io/) offers developer APIs that serve as a conduit for interacting with other services and dependencies and abstract the application from the specifics of those services and dependencies. Dapr and .NET Aspire can work together to improve your local development experience. By using Dapr with .NET Aspire, you can focus on writing and implementing .NET-based distributed applications instead of local on-boarding.
 
-In this guide, you'll learn how to take advantage of Dapr's abstraction and .NET Aspire's opinionated configuration of cloud technologies to build simple, portable, resilient, and secured microservices at-scale on Azure.
+In this guide, you'll learn how to take advantage of Dapr's abstraction and .NET Aspire's opinionated configuration of cloud technologies to build simple, portable, resilient, and secured microservices at scale.
 
-[!INCLUDE [aspire-prereqs](../includes/aspire-prereqs.md)]
+## Comparing .NET Aspire and Dapr
 
-In addition to the prerequisites for .NET Aspire, you will need:
+At first sight Dapr and .NET Aspire may look like they have overlapping functionality, and they do. However, they take different approaches. .NET Aspire is opinionated on how to build distributed applications on a cloud platform and focuses on improving the local development experience. Dapr is a runtime that abstracts away the common complexities of the underlying cloud platform both during development and in production. It relies on sidecars to provide abstractions for things like configuration, secret management, and messaging. The underlying technology can be easily switched out through configuration files, while your code does not need to change.
 
-- Dapr version 1.13 or later
+| Aspect | .NET Aspire | Dapr |
+| --- | --- | --- |
+| Purpose | Designed to make it easier to develop .NET cloud-native solutions on local development computers. | Designed to make it easier to develop and run distributed apps with common APIs that can be easily swapped. |
+| APIs | Developers must call resource APIs using their specific SDKs | Developers call APIs in the Dapr sidecar, which forwards the call to the correct API. It's easy to swap resource APIs without changing code in your microservices. |
+| Languages | You write microservices in .NET languages. | You can call Dapr sidecar functions in any language that supports HTTP/gRPC interfaces. |
+| Security policies | Doesn't include security policies to control communication between microservices and resources. | Includes customizable security policies that control which microservices have access to other services or resources. |
```

Comment:
This is misleading as well, as .NET Aspire can securely configure connections between inter-dependent resources.

Reply:
Fixed.